docs/CmdLineQueriesResults.md

```diff
@@ -11,3 +11,7 @@ The result of each command-line query may be 0 or 1 as followed:
 | --connectivity | ConnectivityMap | the result of this query is always 0 | None |
 | --semantic_diff | SemanticDiff | both sets of NetworkPolicies are semantically equivalent | the given sets of NetworkPolicies are not semantically equivalent |
 
+## A query will not be executed when:
+1. The config/s type is not relevant for performing the query.
+2. An input config does not include NetworkPolicies.
```

Comment:
I think the second case is not accurate.. it's only on permits/forbids query when there given permit/forbid config is empty.